### DISCUSSION TITLE
CSS Combinator selectors does not work in Next.js

### DISCUSSION THREAD

**Discussion by youngnishant** [vercel/next.js#17599] [Help] [answered]
# CSS Combinator selectors does not work in Next.js

## 
div p {
  background-color: yellow;
}

while using CSS some selectors does not work especially combinators such as given above.
## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to index.js
2. use <style jsx> for css
3. write down combinators such as =>.    .link img{}

## Expected behavior

it should work properly according the real css file

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

**Comment by rawat07102** [comment]
you have to surround your styles with backtick \``
<style jsx>
  { `
    div p {
        backround-color: yellow;
    }
  ` }
</style>

**Comment by youngnishant** [comment]
@rawat07102  yeah, it is still not working

**Comment by jamesmosier** [comment, marked as answer]
Hi @youngnishant.

[Check out the styled-jsx docs](https://github.com/vercel/styled-jsx#global-styles) in which it mentions if you want to use global styles you can do it like this:

```
<style jsx global>{`
  div p {
    background: red
  }
`}</style>
```

---

**Edit** Misunderstood the original question. But [see below](https://github.com/vercel/next.js/discussions/17599#discussioncomment-90626) as this should work as expected.

**Reply by youngnishant** [reply]
I'm using Descendant combinator in css, which is still not working.

**Reply by youngnishant** [reply]
Yeah, I went through the jsx styling already, still not working

**Reply by jamesmosier** [reply]
Can you provide a minimal repository that reproduces this?

I just tested this exactly and it worked

![Screen Shot 2020-10-05 at 9 39 59 AM](https://user-images.githubusercontent.com/2854919/95086671-c3afe600-06ee-11eb-8023-ba014e5d29bd.png)

**Comment by rawat07102** [comment]
@youngnishant can you provide code screenshot?

**Comment by youngnishant** [comment]
![Screenshot 2020-10-05 at 1 32 39 PM](https://user-images.githubusercontent.com/47674286/95054379-6ed09780-070f-11eb-9a72-2a4f9cccc672.png)
![Screenshot 2020-10-05 at 1 32 20 PM](https://user-images.githubusercontent.com/47674286/95054382-709a5b00-070f-11eb-87a1-28bbc7366758.png)
![Screenshot 2020-10-05 at 1 32 49 PM](https://user-images.githubusercontent.com/47674286/95054384-71cb8800-070f-11eb-9a45-439ebb85de1d.png)

**Reply by youngnishant** [reply]
@rawat07102 @jamesmosier 
I've uploaded the screenshots of the Problem

**Comment by youngnishant** [comment]
![Screenshot 2020-10-05 at 1 36 35 PM](https://user-images.githubusercontent.com/47674286/95054650-cff86b00-070f-11eb-90d4-72ac024f22d9.png)

that login box should look like this

**Comment by rawat07102** [comment]
works here [https://codesandbox.io/s/infallible-water-qzbbv?file=/pages/index.js](url), it could just be a styling issue
